scraper/src/fetch_practicalplants.js

```diff
@@ -1,76 +1,13 @@
-import { readFileSync, readdirSync, writeFileSync } from 'fs';
+import { readFileSync, readdirSync, writeFileSync, existsSync, mkdirSync } from 'fs';
 import { parse } from 'node-html-parser';
 import { parse as json2csv } from 'json2csv';
 import { config } from 'dotenv';
-import axios from 'axios';
+import { fetchGermanName } from './helpers/helpers.js';
 
 config();
 
 const archivePath = process.env.PRACTICALPLANTSPATH;
 
-function createDistinctFamilyDetails(details) {
-  const detailsCopy = JSON.parse(JSON.stringify(details));
-  const distinctFamilyDetails = [];
-  const distinctFamilyList = [];
-  detailsCopy.forEach((detail) => {
-    const family = detail['Family'];
-    if (!distinctFamilyList.includes(family)) {
-      distinctFamilyList.push(family);
-      Object.keys(detail).forEach((key) => {
-        if (key !== 'Family') {
-          detail[key] = null;
-        }
-      });
-      distinctFamilyDetails.push(detail);
-    }
-  });
-  return distinctFamilyDetails;
-}
-
-function createDistinctGenusDetails(details) {
```

Comment:
Genus entries (column genus) are sometimes missing, they can be extracted from unique_name. Also here: entries for every genus should end up in plants.csv (and plants table).

Reply:
sure, I'll add this to the leftover ticket. I've noticed some individual problems caused by the hierarchy naming, still not sure if it is applicable to every plant in the database

Reply:
Hierarchy and enums/ints are the most important goals. I would like to avoid that someone else needs to dive into your scraper, which would take lots of time.